boards/nxp/imx943_evk/board.cmake

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright 2025 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_SOC_MIMX94398_A55)
+
+board_runner_args(jlink "--device=MIMX94398_A55" "--no-reset" "--flash-script=${CMAKE_BINARY_DIR}/zephyr/runner.jlinkscript")
+
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
+
```

Comment:
indent with 2 spaces, remove empty lines on line 11 and 7

Reply:
Fixed, thanks.